src/gui/dialogs/campaign_difficulty.cpp

```diff
@@ -99,10 +101,29 @@ void campaign_difficulty::pre_show(window& window)
 			list.select_last_row();
 		}
 
-		widget* widget = grid.find("victory", false);
-		if(widget && !preferences::is_campaign_completed(campaign_id_, d["define"])) {
-			widget->set_visible(widget::visibility::hidden);
+		styled_widget& widget = find_widget<styled_widget>(&grid, "victory", false);
+		if(preferences::is_campaign_completed(campaign_id_, d["define"])) {
+			// Use different laurels according to the difficulty level, following the
+			// pre-existing convention established in campaign_selection class.
+			// Assumes ascending order of difficulty.
+			if(difficulty_count == 0) {
+				widget.set_label(game_config::images::victory_laurel_easy);
+			}
+
+			else if(difficulty_count + 1 >= difficulty_max) {
```

Comment:
The line spacing for the else blocks looks a bit odd here and below. Is there a reason for that?

Reply:
No particular reason other than I haven't really done C++ development in a long time, mostly Python recently, and I'm probably influence by its 'sparse is better than dense mandate. If it was up to me, I would put the opening brace on its own line but I was at least trying to follow existing convention there. Since the rest of the code here uses dense formatting, I'll follow that.